### PR TITLE
Feature/mms 859

### PR DESCRIPTION
**Pull request by dlamoris**

### CODE REVIEW

File: mms-ent/migration/3.2.0/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+2. Run `update_commits.py [your_admin_username_for_alfresco] [your_ip_host_name]`
+
+### Converting indexes from 'mms' to 'projectIds'
+
+3. Run the template mapping script without the delete all indexes.  Make sure not to delete indexes if it's the same server.
+4. Run `convert_indexes.py [src_hostname] [your_admin_username_for_alfresco] [destination_ElasticSearch_hostname]`
+
+### Notes on Failure
+
+If the converting_indexes.py script fails you'll either have to remove all the indexes and start over or remove the last project it ran and edit the script to start there.  To remove a project run `curl -XDELETE 'http://[hostname]:9200/[projectId]'`.
```
Comment (dlamoris) [inline mms-ent/migration/3.2.0/README.md]: projectId is lowercase in the index

File: mms-ent/migration/3.2.0/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+### Database Update
+
+We had to remove some of the edges being tracked in the graph.
+
+1. Run `update_database.py [your_database_name] [your_ip_host_name] [your_database_user_name]`
+
+### Updating Commit, Project and Ref Obejct with the _ProjectId key
+
+2. Run `update_commits.py [your_admin_username_for_alfresco] [your_ip_host_name]`
```
Comment (dlamoris) [inline mms-ent/migration/3.2.0/README.md]: these scripts rely on the 3.1 rest api, so mms needs to be up

File: mms-ent/migration/3.2.0/update_commits.py [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+@author: Laura Mann
+
+Updates the commits, projects and ref objects with the correct project Id key
+
+'''
+
+
+def main(args):
+    es = Elasticsearch([{'host': args[1], 'port': 9200}], timeout=300)
+    util.base_url = args[1] + ":8080"
```
Comment (dlamoris) [inline mms-ent/migration/3.2.0/update_commits.py]: should prepend http?

File: mms-ent/migration/3.2.0/update_commits.py [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+    for project in projects:
+        projectId = project['id']
+        project_elastic[project['id']] = project['_elasticId']
+        project_ids.append(projectId)
+    return set(project_ids), project_elastic
+
+
+if __name__ == '__main__':
+    if len(sys.argv) < 3:
+        print "You need to pass the username and the host"
```
Comment (dlamoris) [inline mms-ent/migration/3.2.0/update_commits.py]: order is host and alfresco admin username

File: mms-ent/migration/3.2.0/update_database.py [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+        rows = cur.fetchall()
+        for row in rows:
+            orgId = str(row[0])
+            orgNames = str(row[1])
+        if orgNames is not None:
+            print(orgId)
+            print(orgNames)
+            cur.execute('DELETE FROM edges WHERE parent = %s', [orgId])
+        # CASE 2: holding_bin or holding_bin_{orgid}
+        holding_bins = [orgNames, 'holding_bin', 'holding_bin_' + orgNames]
```
Comment (dlamoris) [inline mms-ent/migration/3.2.0/update_database.py]: these lines should be inside the if statement

File: mms-ent/migration/3.2.0/update_commits.py (hunk 3 above)
Comment (lauraerinmann) [reply mms-ent/migration/3.2.0/update_commits.py]: I think it makes more sense for the user to decide, it could be https...

**Pull request merged**